domains/transfer/Shared/ProjectOriginClients/ClientCredentialsService.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+
+namespace ProjectOriginClients;
+
+
+public interface IClientCredentialsService
+{
+    Task<string> GetAccessTokenAsync();
+}
+
+public class ClientCredentialsOptions
+{
+    public const string Prefix = "ClientCredentials";
```

Comment:
How do we want our appsettings to look?

ClientCredentials__ClientId
ClientCredentials__ClientSecret
...... ?